W-10877309 Exchange create custom policy for Flex Gateway jh (#188)
mulesoft/docs-exchange: modules/ROOT/pages/to-create-an-asset.adoc

* Mule applications are published to Exchange from Anypoint Runtime Fabric and managed in Runtime Fabric, and are not visible in the Exchange user interface.
* External libraries are published to Exchange from the xref:to-publish-assets-maven.adoc[Exchange Maven Facade API] and managed with this API, and are not visible in the Exchange user interface.

Exchange fully supports RAML fragments. Currently, Exchange does not support importing and managing OAS 3.0 components as Exchange dependencies inside a specification.

> This illustration summarizes how some asset types (in green) appear in Exchange:

Reviewer: ```suggestion
This illustration summarizes how some asset types appear in Exchange:
```

. (Optional) Select an option for *Lifecycle state*. The default is *Stable*.
. Select *Publish*.

> == Create Custom Policy Definition Asset 

Reviewer: ```suggestion
== Create a Custom Policy Definition Asset 
```

> Policies enforce regulations to help manage security, control traffic, and improve API adoption. For example, a policy can control authentication, access, and service level agreement (SLA). You can create a policy definition based on your specific business requirements. Develop custom policy definitions and apply them to APIs to extend existing functionality or define new ones.

Reviewer: ```suggestion
Policies enforce regulations to help manage security, control traffic, and improve API adoption. For example, a policy can control authentication, access, and service level agreements (SLAs). You can create a policy definition based on your specific business requirements. Develop custom policy definitions and apply them to APIs to extend existing functionality or define new functionality.
```

To publish a custom policy definition:

. In Exchange, click *Publish new asset*.
. Enter the asset name.
. Select the *Policy* asset type from the drop-down list.
. Click *Choose File* to select a JSON schema file.
. Click *Choose File* to select a YAML file.
. To edit the GroupId, AssetId, Version, and API version, click *Advanced*. 
+
Exchange generates the group ID, asset ID, and version. You can change these values. You can also change an API asset’s version and API version separately. The advanced settings are most often used to change the asset version.

> . From Lifecycle state, select *Stable*. 

Reviewer: the Lifecycle state

+
The policy must be in the Stable state to upload the implementation file in a later step.
. Select *Publish*. 
+

> *Pending* status appears next to *Implementations* in the left navigation bar.

Reviewer: ```suggestion
The *Pending* status appears next to *Implementations* in the navigation bar.
```
